kornia/models/_hf_models/onnx_community.py

```diff
@@ -4,42 +4,122 @@
 from typing import Any, Optional
 
 from kornia.core import ImageSequential
-from kornia.onnx.utils import ONNXLoader
-from kornia.onnx.sequential import ONNXSequential
+from kornia.onnx.utils import ONNXLoader, io_name_conversion
+from kornia.onnx import ONNXSequential, load, add_metadata
 from kornia.config import kornia_config
-
 from kornia.core.external import onnx
+from kornia.models.base import ModelBaseMixin
 from .preprocessor import PreprocessingLoader
 
 
 class ONNXComunnityModelLoader:
 
     def __init__(
-        self, model_name: str, model_type: str = "model", preprocessing: bool = True, cache_dir: Optional[str] = None
+        self, model_name: str, model_type: str = "model", cache_dir: Optional[str] = None
     ) -> None:
         self.model_name = model_name
         self.model_type = model_type
-        self.preprocessing = preprocessing
         
         if cache_dir is None:
             cache_dir = kornia_config.hub_onnx_dir
         self.loader = ONNXLoader(cache_dir=os.path.join(cache_dir, self.model_name))
+        self.model_url = f"https://huggingface.co/onnx-community/{self.model_name}/resolve/main/onnx/{self.model_type}.onnx"
```

Comment:
Does it mean that this loader only works for hugging face ? Can it be generic? Otherwise maybe rename de class name